### PR TITLE
Support configure chromedriver cdn url via npm config property

### PR DESCRIPTION
**Pull request by fengmk2**
```shell
npm install phantomjs --chromedriver_cdnurl=http://npm.taobao.org/mirrors/chromedriver
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,27 @@ The package has been set up to fetch and run ChromeDriver for MacOS (darwin),
...
+Or add property into your [`.npmrc`](https://docs.npmjs.com/files/npmrc) file.
+
+```
+chromedriver_cdnurl=http://npm.taobao.org/mirrors/chromedriver
+```
+
+Another option is to use PATH variable `CHROMEDRIVER_CDNURL`.
+
+```shell
+CHROMEDRIVER_CDNURL=http://npm.taobao.org/mirrors/chromedriver npm install phantomjs
```
Comment (giggio) [inline README.md]: Phantomjs?

**Comment by fengmk2** [comment]
@giggio typo fix

**Comment by giggio** [comment]
Published to npm as 2.14.1.

**Comment by fengmk2** [comment]
Awesome!

**Pull request merged**